src/dmtcpplugin.cpp

```diff
@@ -309,9 +305,7 @@ dmtcp_get_executable_path(void)
 EXTERNC const char *
 dmtcp_get_uniquepid_str(void)
 {
-  static string *uniquepid_str = NULL;
-
-  uniquepid_str =
+  static string *uniquepid_str =
     new string(UniquePid::ThisProcess(true).toString());
```

Comment:
Doesn't this create a memory leak by allocating in the heap, instead of tmpdir on the stack?